### DISCUSSION TITLE
Add an automatic field cleaning functionality to trim spaces from left and right of strings

### DISCUSSION THREAD

**Discussion by vincentdev15** [livewire/livewire#5854] [3. Feature ideas]
It would be pratice to have an automatic field cleaning functionality to trim spaces from left and right of strings, like the TrimStrings middleware.

**Comment by joshhanley** [comment]
@vincentdev15 the problem with that is, that a user could be typing say their full name into an input field (assuming using `wire:model.live`) and it would strip the space out as they are typing it. Hence why Livewire doesn't implement anything like that. That's something you will need to do yourself.
